earth_enterprise/src/common/packet.h

```diff
@@ -449,14 +451,14 @@ struct etPointerTranslator
 #ifdef FORCE_BASE32
 struct etStreetPacketData_BASE32
 {
-  uint32           name;
+  std::uint32_t           name;
   int          texId_deprecated;
   ushort       numPt;
   ushort       bitFlags;
 #ifdef VECTOR_COMPRESSION
-  uint32 localPt;
+  std::uint32_t localPt;
 #else
-  uint32 localPt;
+  std::uint32_t localPt;
 #endif
```

Comment:
can probably just remove the guards here since it looks like `localPt` will always be declared